src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -105,147 +91,6 @@ private EventPipeController()
             // Set the config file path.
             // BaseDirectory could be null, in which case this could throw, but it will be caught and ignored: https://github.com/dotnet/coreclr/issues/24053
             m_configFilePath = Path.Combine(AppContext.BaseDirectory!, BuildConfigFileName());
```

Comment:
Is there any reason we're leaving this here when we're deleting all its uses?

Reply:
I missed it. Thanks!

Reply:
It's also in line 51 in case you missed that too :-)